docs/JestObjectAPI.md

```diff
@@ -908,6 +908,15 @@ This function is not available when using legacy fake timers implementation.
 
 ## Misc
 
+### `jest.getSeed()`
+Every time Jest runs a seed value is randomly generated which you could use in a pseudorandom number generator or anywhere else.
```

Comment:
```suggestion
### `jest.getSeed()`

Every time Jest runs a seed value is randomly generated. It can be used in a pseudorandom number generator or anywhere else.
```